probing/retokenize_edge_data.py

```diff
@@ -75,7 +75,7 @@ def _get_bert_tokenizer(model_name, do_lower_case):
 # of a TokenAligner instance and a list of tokens.
 def align_moses(text: Text) -> Tuple[retokenize.TokenAligner, List[Text]]:
     moses_tokens = MosesTokenizer.tokenize(text)
-    cleaned_moses_tokens = utils.unescape_moses(moses_tokens)
+    cleaned_moses_tokens = MosesTokenizer.detokenize(moses_tokens)
```

Comment:
It's not clear if this behaves the same as the old version. Can we keep the old `utils.unescape_moses` here until after the ACL deadline at least?

Reply:
Sure - the detokenize function is a renaming of unescape_moses,  the function is the same, but I'll keep the old utils.unescape_moses as well.